api/src/main/java/ca/bc/gov/educ/grad/report/api/util/LogHelper.java

```diff
@@ -72,7 +76,8 @@ public void logServerHttpReqResponseDetails(@NonNull final HttpServletRequest re
      * @param responseCode
      * @param correlationID
      */
-    public void logClientHttpReqResponseDetails(@NonNull final HttpMethod method, final String url, final int responseCode, final List<String> correlationID, final boolean logging) {
+    public void logClientHttpReqResponseDetails(@NonNull final HttpMethod method, final String url, final int responseCode, final List<String> correlationID,
```

Comment:
Please ensure this method is being called by log() method in ReportServiceConfiguration.class. It is currently not used.